buildkite/resource_pipeline.go

```diff
@@ -152,7 +152,8 @@ func resourcePipeline() *schema.Resource {
 				Type:     schema.TypeString,
 			},
 			"steps": {
-				Required: true,
+				Optional: true,
+				Default: "    steps:\n    - label: ':pipeline: Pipeline Upload'\n      command: buildkite-agent pipeline upload\n",
```

Comment:
I'd hoped it would be simple 👍
I wonder if using a `const` and/or multiline string would be nicer https://www.calhoun.io/6-tips-for-using-strings-in-go/

Reply:
Shall check that option out - TY @jradtilbrook 👍

Reply:
Changed to const, file scope